braindecode/datasets/base.py

```diff
@@ -217,18 +217,16 @@ class BaseConcatDataset(ConcatDataset):
     ----------
     list_of_ds : list
         list of BaseDataset, BaseConcatDataset or WindowsDataset
-    seq_target_transform : callable | None
-        Function to call on sequences of targets before returning them. Only
-        called if the dataset is indexed with lists of indices to return
-        sequences.
+    target_transform : callable | None
+        Optional function to call on targets before returning them.
     """
-    def __init__(self, list_of_ds, seq_target_transform=None):
+    def __init__(self, list_of_ds, target_transform=None):
```

Comment:
We had some longer discussion about this already, that we may want to allow different transforms in the future for single items or sequences. Or at least that changing to one transform that is always applied may break unexpectedly, like if you have a transform that deals with sequences of labels and then maybe this works:
```
dataset[0:3]
```
whereas this will throw some error because the supplied transform won't handle single items well:
```
dataset[0]
```

Therefore we first keep like this make explicit this is transformation applied to sequences and this allows user still to retrieve original single-item labels untransformed